src/patternfly/components/Label/examples/Label.md

```diff
@@ -596,24 +603,36 @@ This style of label is used to indicate overflow within a label group.
 {{#> label label--id="compact-editable-label-active" label--modifier="pf-m-compact pf-m-blue" label--IsEditable="true" label--IsEditableActive="true"}}Compact editable active{{/label}}
 ```
 
+### Add label
+
+This style of label is used to add new labels to a label group.
+
+```hbs isBeta
+{{#> label label--id="add" label--IsAdd="true"}}
+  Add Label
+{{/label}}
+```
+
 ## Documentation
 
 ### Usage
-| Class | Applied to | Outcome |
-| -- | -- | -- |
-| `.pf-c-label` | `<span>`, `<button>` | Initiates a label. Without a color modifier, the label's default style is grey. Use a color modifier to change the label color. Use a `<button>` if the label is an overflow label used in the label group. **Required** |
-| `.pf-c-label__content` | `<span>`, `<a>`, `<button>` | Iniates a label content. Use as an `<a>` element if the label serves as a link. Use a `<button>` if the label serves as an action. **Required** |
-| `.pf-c-label__icon` | `<span>` | Initiates a label icon. |
-| `.pf-c-label__text` | `<span>` | Initiates label text. |
-| `.pf-c-label__editable-text` | `<button>`, `<input>` | Initiates editable label text. See the [editable](#editable) example for details about handling behavior in Javascript.|
-| `.pf-m-outline` | `.pf-c-label` | Modifies label for outline styles. |
-| `.pf-m-compact` | `.pf-c-label` | Modifies label for compact styles. |
-| `.pf-m-overflow` | `.pf-c-label` | Modifies label for overflow styles for use in a label group. |
-| `.pf-m-blue` | `.pf-c-label` | Modifies the label to have blue colored styling. |
-| `.pf-m-green` | `.pf-c-label` | Modifies the label to have green colored styling. |
-| `.pf-m-orange` | `.pf-c-label` | Modifies the label to have orange colored styling. |
-| `.pf-m-red` | `.pf-c-label` | Modifies the label to have red colored styling. |
-| `.pf-m-purple` | `.pf-c-label` | Modifies the label to have purple colored styling. |
-| `.pf-m-cyan` | `.pf-c-label` | Modifies the label to have cyan colored styling. |
-| `.pf-m-editable` | `.pf-c-label` | Modifies label for editable styles. |
-| `.pf-m-editable-active` | `.pf-c-label.pf-m-editable` | Modifies editable label for active styles. |
+
+| Class                        | Applied to                  | Outcome                                                                                                                                                                                                                  |
```

Comment:
Nit - this kind of spacing is hard to maintain manually when making updates to the docs - or everyone making changes will need the same formatter - so we just use single spaces in the docs tables to keep it simple for all contributors.

Reply:
Ah I think maybe Prettier got to this before I disabled it for the workspace. I noticed the SCSS file also had some formatting changed but turned Prettier off and reverted that, this one snuck by me.